assets/src/edit-story/components/form/dateTime/dateTime.js

```diff
@@ -40,13 +45,19 @@ const DateTimeWrapper = styled.div`
   padding: 4px;
 `;
 
+const StyledButton = styled(Button)`
+  color: #006edc;
```

Comment:
Using the colour from the calendar here for consistency. We can update this to the design theme when redesigning the calendar.